x-pack/legacy/plugins/lens/public/indexpattern_datasource/indexpattern_suggestions.ts

```diff
@@ -196,30 +196,29 @@ function addFieldAsMetricOperation(
     suggestedPriority: undefined,
     field,
   });
-  const newColumnId = generateId();
+  const addedColumnId = generateId();
 
-  const [, metrics] = separateBucketColumns(layer);
+  const [buckets, metrics] = separateBucketColumns(layer);
```

Comment:
this is never read, is it?

Reply:
Not any more, good catch- I was going to rewrite the buckets in an earlier iteration.